CSharpAdvanced/Peony.cs

```diff
@@ -0,0 +1,6 @@
+namespace CSharpAdvanced;
+
+public class Peony : Flower
```

Comment:
It is okay for this HW, but in future if you see an empty class in your project - it is a marker that something is not okay with your architecture :)